fractal_tasks_core/cellpose_segmentation.py

```diff
@@ -167,6 +384,8 @@ def cellpose_segmentation(
     min_size: int = 15,
     augment: bool = False,
     net_avg: bool = False,
+    use_masks: bool = False,
+    primary_label_name: Optional[str] = None,
```

Comment:
Is that actually still used? It's commented out on line 715

Reply:
Agreed, fixed